examples/angular/simple/src/simple-example.component.ts

```diff
@@ -0,0 +1,43 @@
+import { AngularQueryDevtoolsComponent } from '@tanstack/angular-query-devtools-experimental'
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core'
+import { CommonModule } from '@angular/common'
+import { CreateQuery } from '@tanstack/angular-query-experimental'
+import axios from 'axios'
+
+type Response = {
+  name: string
+  description: string
+  subscribers_count: number
+  stargazers_count: number
+  forks_count: number
+}
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'simple-example',
+  standalone: true,
+  template: `
+    <ng-container *ngIf="query().isPending">Loading...</ng-container>
+    <ng-container *ngIf="query().error as error">
+      An error has occurred: {{ error?.message }}
+    </ng-container>
+    <div *ngIf="query().data as data">
+      <h1>{{ data.name }}</h1>
+      <p>{{ data.description }}</p>
+      <strong>👀 {{ data.subscribers_count }}</strong>
+      <strong>✨ {{ data.stargazers_count }}</strong>
+      <strong>🍴 {{ data.forks_count }}</strong>
+    </div>
+    <angular-query-devtools initialIsOpen />
+  `,
+  imports: [AngularQueryDevtoolsComponent, CommonModule],
+})
+export class SimpleExampleComponent {
+  query = inject(CreateQuery)(() => ({
+    queryKey: ['repoData'],
+    queryFn: () =>
```

Comment:
In an Angular world, I'd like to see the queryFn accept an observable rather than a promise (or both), because most of the time this will be in a service using Angular's http client instead of axios, because of interceptors and better SSR support in general.

Reply:
Agree, it's not hard to convert an observable to a promise but it's what Angular developers would commonly expect to just work out of the box.

Reply:
Put on my todo list, with high prio. I do think it's important to support both promises and observables.